velox/exec/fuzzer/WriterFuzzer.cpp

```diff
@@ -422,6 +506,43 @@ void WriterFuzzer::verifyWriter(
       assertEqualResults(reference_data, {actual}),
       "Velox and reference DB results don't match");
 
+  // 4. Verifies sorting.
+  if (isSorted(partitionKeys.size(), bucketCount, sortBy.size())) {
+    const std::vector<std::string> sortColumnNames = {
+        names.begin() + sortColumnOffset,
+        names.begin() + sortColumnOffset + sortBy.size()};
+    const std::vector<TypePtr> sortColumnTypes = {
+        types.begin() + sortColumnOffset,
+        types.begin() + sortColumnOffset + sortBy.size()};
+
+    // Read from each file and check if data is sorted as presto sorted result.
+    for (const auto& split : splits) {
+      auto readSingleSplitPlan =
+          PlanBuilder()
+              .tableScan(
+                  generateOutputType(sortColumnNames, sortColumnTypes, 0, 0))
+              .planNode();
+      auto singleSplitData = execute(readSingleSplitPlan, 1, {split});
+
+      auto singleSplitSql = sortSql(
```

Comment:
Shall we move sortSql to reference DB API? thanks!

Reply:
Most of the logic is more about converting split information to sql, which is not applicable to move to referenceDB API, so I will keep it here